lib/topological_inventory/ingress_api/messaging_client.rb

```diff
@@ -31,11 +29,7 @@ def with_messaging_client
         end
 
         def messaging_client
-          Thread.current[:messaging_client]
```

Comment:
I know you ask why in the description. Is there any clue between @lindgrenj6 @roliveri @slemrmartin why this wasn't an issue before?

Reply:
![image](https://user-images.githubusercontent.com/5856504/97185992-be364080-176e-11eb-93c8-e820f6910677.png)

So after checking out the commit before the rdkafka changes - it's doing the same thing still. 

I'm wondering if it has to do with the fact that rdkafka is a FFI library vs ruby-kafka which is pure ruby, maybe the ruby-kafka instances were gc'd fine, whereas FFI leaves a thread open and running?